src/tools/rust-analyzer/Cargo.lock

```diff
@@ -888,9 +888,9 @@ checksum = "e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646"
 
 [[package]]
 name = "libc"
-version = "0.2.148"
+version = "0.2.150"
```

Comment:
Is this relevant and necessary?

Reply:
*(wow you are too fast, didn't have time to fully write the comment explaining this change)*

Yes, it includes https://github.com/rust-lang/libc/pull/3410 which was published in 0.2.150